build/notarize.js

```diff
@@ -5,14 +5,15 @@
  */
 
 require('dotenv').config();
-const { notarize } = require('electron-notarize');
+const { notarize } = require('@electron/notarize');
 const pkgJson = require('../package.json');
 
 exports.default = ({ electronPlatformName, appOutDir }) =>
     electronPlatformName === 'darwin'
         ? notarize({
               appBundleId: pkgJson.build.appId,
               appPath: `${appOutDir}/${pkgJson.build.productName}.app`,
+              teamId: 'P3R8YQEV4L',
```

Comment:
To my understanding the team-ID isn't secret, so we _could_ keep it in here. But it would be nicer to turn it into an (open) variable just like the Apple-ID, to handle both in the same way.